tests/Collection/CrudSpecFunctionalTest.php

```diff
@@ -87,7 +88,7 @@ public function provideSpecificationTests()
         $testArgs = [];
 
         foreach (glob(__DIR__ . '/spec-tests/*/*.json') as $filename) {
-            $json = json_decode(file_get_contents($filename), true);
+            $json = json_decode(file_get_contents($filename), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
Noted that `512` is the default value for `$depth`.

Reply:
Yes, I believe that the `assertIsObject` check later in this file tripped rector up a bit.